### PR TITLE
Fail format.sh if yapf/flake8 versions are incorrect.

### PR DESCRIPTION
**Pull request by robertnishihara**
We could do the same thing for `clang-format` as @stephanie-wang mentioned.

### CODE REVIEW

**Review by ericl — APPROVED** [review]
this is a lifesaver

**Comment by stephanie-wang** [comment]
Do you think we could do the same with `clang-format -version`?

**Comment by robertnishihara** [comment]
@stephanie-wang done.

**Comment by stephanie-wang** [comment]
```
travis_time:start:052dae94
[0K$ ./ci/travis/format.sh --all
Please install clang-format 6.0.0. You currently are using 7.0.0.
travis_time:end:052dae94:start=1562150104647393684,finish=1562150104928984368,duration=281590684
[0K[31;1mThe command "./ci/travis/format.sh --all" exited with 1.[0m
```

Looks like we should allow multiple versions.

File: .travis.yml [hunk 1]
```diff
@@ -38,7 +38,6 @@ matrix:
       env: LINT=1 PYTHONWARNINGS=ignore
       before_install:
         - sudo apt-get update -qq
-        - sudo apt-get install -qq clang-format-6.0
```
Comment (robertnishihara) [inline .travis.yml]: The version of `clang-format` installed here is actually `clang-format-6.0`, which wasn't used. It seems that `clang-format` version `7.0.0` comes with the base image.

**Pull request merged**